### PR TITLE
EZP-29348: Edit buttons in tables are not aligned with delete button in table header

### PR DESCRIPTION
**Pull request by tischsoic**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29348
| Bug fix?      | -
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | -
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Edit buttons in tables are not aligned with delete button in table header.

**For QA** : check pages which are rendered using modified twigs. See list in _Files Changed_ on GitHub. 

Effect:
screen shot 2018-06-28 at 13 41 41

#### Checklist:
- [ ] ~Coding standards (`$ composer fix-cs`)~
- [x] Ready for Code Review

Edited tables:
screen shot 2018-08-08 at 11 27 46
screen shot 2018-08-08 at 11 30 07
screen shot 2018-08-08 at 11 32 33
screen shot 2018-08-08 at 11 33 13
screen shot 2018-08-08 at 11 33 46
screen shot 2018-08-08 at 11 34 13
screen shot 2018-08-08 at 11 34 58
screen shot 2018-08-08 at 11 35 42
screen shot 2018-08-08 at 11 36 48
screen shot 2018-08-08 at 11 37 24
screen shot 2018-08-08 at 11 39 09
screen shot 2018-08-08 at 11 39 33
screen shot 2018-08-08 at 11 40 06
screen shot 2018-08-08 at 11 40 31
screen shot 2018-08-08 at 11 40 55
screen shot 2018-08-08 at 11 41 30
screen shot 2018-08-08 at 11 42 28
Dashboard:
screen shot 2018-08-08 at 11 43 30
Dashboard:
screen shot 2018-08-08 at 11 43 53
screen shot 2018-08-08 at 11 44 20
screen shot 2018-08-08 at 11 44 33
screen shot 2018-08-08 at 11 44 43
screen shot 2018-08-08 at 11 45 25

screen shot 2018-08-20 at 12 02 10
screen shot 2018-08-20 at 12 01 33
screen shot 2018-08-20 at 12 01 12
screen shot 2018-08-20 at 11 59 37

### CODE REVIEW

**Comment by lserwatka** [comment]
@tischsoic what is needed her? I would like to move this forward.

**Comment by webhdx** [comment]
That's quite a lot of screenshots. 

image

**Comment by tischsoic** [comment]
@webhdx ask @sunpietro why we need them all. 😄

**Comment by webhdx** [comment]
That's quite a lot of screenshots. 

giphy

**Comment by sunpietro** [comment]
ping @inakijv

**Comment by tischsoic** [comment]
Ping @sunpietro @dew326 :)

**Comment by lserwatka** [comment]
@mnocon @m-tyrala Behat is failing here. Is this because of this change.

**Comment by mnocon** [comment]
@lserwatka yes, looks like this PR changes some of the classes we're using in selectors (`ez-checkbox-cell` no longer exists from what I see).

Can we handle it later this week, once we're finished with the certification?

**Comment by lserwatka** [comment]
Yes, not urgent.

File: src/bundle/Resources/views/admin/role/list.html.twig [hunk 1]
```diff
@@ -74,25 +74,25 @@
...
-                    <td>{{ role.id }}</td>
-                    <td class="text-right">
+                    <td class="ez-table__cell">{{ role.id }}</td>
+                    <td class="ez-table__cell ez-table__cell--has-action-btns text-right">
                         {% if can_assign %}
                             <a
                                 title="{{ 'role.view.list.action.assign_to_users_or_groups'|trans|desc('Assign to Users/Groups') }}"
                                 href="{{ path('ezplatform.role_assignment.create', { roleId: role.id }) }}"
-                                class="btn btn-icon mx-3">
+                                class="btn btn-icon mx-2">
```
Comment (inakijv) [inline src/bundle/Resources/views/admin/role/list.html.twig]: I believe that maybe we should keep this as `mx-3` in order to be consistent with the offset defined in `src/bundle/Resources/views/admin/section/list.html.twig` line 97.

**Comment by tischsoic** [comment]
Ping @inakijv - I fixed margin in admin/roles/list.html.twig.

**Comment by tischsoic** [comment]
@mnocon I will check what causes it.

**Comment by lserwatka** [comment]
@mnocon @tischsoic was Behat failure fixed here?

**Comment by mnocon** [comment]
@lserwatka yes, I've made adjustment, the next build should be green

**Comment by lserwatka** [comment]
@tischsoic could you rebase? @mnocon Behat is still red here.

**Comment by mnocon** [comment]
@lserwatka yes, sadly this is the same Travis issue I've talked about during daily (so far I've summarised what I've found in https://github.com/ezsystems/ezplatform/pull/319#issuecomment-414662570)

**Comment by lserwatka** [comment]
@tischsoic could you rebase here, Behat should be passing now.

**Comment by tischsoic** [comment]
@lserwatka @mnocon rebased

**Comment by mnocon** [comment]
@tischsoic is it intended that the link in link detail view is moved to the right?
screen shot 2018-08-27 at 10 19 55
That looks strange to me.
Note: Travis should be passing now 🎉

**Comment by tischsoic** [comment]
@mnocon It should not be moved to right, I will fix it. :)

**Pull request merged**